aten/src/ATen/core/boxing/KernelFunction_test.cpp

```diff
@@ -263,7 +263,7 @@ void expectOutOfPlaceBoxedCallingWorks(const KernelFunction& func) {
   auto s = 1.0f;
   auto t = at::zeros({1});
   vector<IValue> stack {s, t};
-  func.callBoxed(dummy, &stack);
+  func.callBoxed(dummy, c10::DispatchKeySet(), &stack);
```

Comment:
Passing empty dispatch key set here is kind of questionable, not because the test wouldn't pass (it does, obviously), but because it isn't really realistic for the stack here; you'd expect the key set to include CPU. Maybe it would be slightly better to have some sort of placeholder dispatch key set, with a comment saying that it's meaningless and the tests don't rely on it. In this particular case the harm is not great; the harm comes if people copy paste filling in empty DispatchKeySet in situations where they *are* obligated to populate it properly.

Reply:
Yes, thanks. Definitely deserves a note + I'll pick a different set that's more semantically correct (like a set of just CPU)